packages/react-components/react-select/src/stories/SelectControlled.stories.tsx

```diff
@@ -1,21 +1,39 @@
 import * as React from 'react';
 import { Select } from '../index';
 import { useId } from '@fluentui/react-utilities';
+import { makeStyles } from '@griffel/react';
+
+const useStyles = makeStyles({
+  root: {
+    maxWidth: '400px',
+
+    // Stack the label above the field with a 2px gap
+    '> label': {
+      display: 'block',
+      marginBottom: '2px',
```

Comment:
I'd recommend you use a vertical spacing token so the story better demonstrates best practices.